docs/Technical-Documentation/Zap-Scan-HTML-Report.md

```diff
@@ -1,26 +1,26 @@
-# OWASP Zap Scan HTML Report

```

Comment:
Is there a way to get this file out of this PR? Don't want to reintroduce our ZAP problem

Reply:
+1